fixtures/futures/tests/bindings/test_futures.swift

```diff
@@ -232,4 +232,33 @@ Task {
 	counter.leave()
 }
 
+// Test a future that uses a lock and that is cancelled.
+counter.enter()
+Task {
+	let task = Task {
+	    try! await useSharedResource(options: SharedResourceOptions(releaseAfterMs: 100, timeoutMs: 1000))
+	}
```

Comment:
Are we sure the `Task` runs the code inside the block as soon as it's built? Or should we have a `await task.value` to ensure the `Task` runs?

Reply:
The task is scheduled as soon as it's built and should run during the `Task.sleep()` call below.  I don't think we can `await` the task itself, since we want to cancel it.

Reply:
OK!